src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/FindStatisticsPageNew.test.tsx

```diff
@@ -180,270 +200,282 @@ describe('FindStatisticsPageNew', () => {
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.queryByText('There are no matching results.'),
+  //   ).not.toBeInTheDocument();
+  // });
+
+  // test('renders correctly when filtered by theme and has no results', () => {
+  //   render(
+  //     <FindStatisticsPageNew
+  //       paging={{ ...testPaging, totalPages: 0, totalResults: 0 }}
+  //       publications={[]}
+  //       query={{ releaseType: 'ExperimentalStatistics' }}
+  //       themes={testThemeSummaries}
+  //     />,
+  //   );
+
+  //   expect(
+  //     screen.getByRole('heading', { name: '0 results' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(screen.getByText('0 pages, filtered by:')).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByRole('button', {
+  //       name: 'Remove filter: Experimental statistics',
+  //     }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByRole('button', { name: 'Clear all filters' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByText('There are no matching results.'),
+  //   ).toBeInTheDocument();
+
+  //   expect(screen.queryByTestId('publicationsList')).not.toBeInTheDocument();
+  // });
+
+  // test('renders correctly with all filters and search', () => {
+  //   render(
+  //     <FindStatisticsPageNew
+  //       paging={{ ...testPaging, totalPages: 1, totalResults: 1 }}
+  //       publications={[testPublications[1], testPublications[2]]}
+  //       query={{
+  //         releaseType: 'AdHocStatistics',
+  //         search: 'find me',
+  //         themeId: 'theme-1',
+  //       }}
+  //       themes={testThemeSummaries}
+  //     />,
+  //   );
+
+  //   expect(
+  //     screen.getByRole('heading', { name: '1 result' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(screen.getByText('Page 1 of 1, filtered by:')).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByRole('button', { name: 'Remove filter: find me' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByRole('button', { name: 'Remove filter: Ad hoc statistics' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByRole('button', { name: 'Remove filter: Theme 1' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.getByRole('button', { name: 'Clear all filters' }),
+  //   ).toBeInTheDocument();
+
+  //   const publicationsList = within(screen.getByTestId('publicationsList'));
+  //   expect(publicationsList.getAllByRole('listitem')).toHaveLength(2);
+
+  //   expect(
+  //     publicationsList.getByRole('heading', { name: 'Publication 2' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     publicationsList.getByRole('heading', { name: 'Publication 3' }),
+  //   ).toBeInTheDocument();
+
+  //   expect(
+  //     screen.queryByText('There are no matching results.'),
+  //   ).not.toBeInTheDocument();
+  // });
 });
+
+// Can't set using the common render to work, so doing it here instead.
```

Comment:
We'll want to sort this out so that we can use the common `render` function as it's pre-configured for use in tests.

If it's issues around using the same instance of `react-query`, due to how the project's currently setup, the `react-query` used as a dependency in common is not going to be the exact same one used in frontend.

To fix this, we'll need to add a `moduleNameMapper` to the Jest config in the frontend's `package.json`.

Reply:
Sorted now, thanks for the tip about adding a moduleNameMapper for react-query.

The tests all pass but I'm getting a warning on them, `It looks like you're using the wrong act() around your test interactions.`, related to next/link which I can't figure out how to get rid of. Have you come across this before?